src/main/java/codegym/tequila/fisioapp/controller/TherapistController.java

```diff
@@ -0,0 +1,27 @@
+package codegym.tequila.fisioapp.controller;
+
+import codegym.tequila.fisioapp.dto.TherapistDto;
+import codegym.tequila.fisioapp.service.TherapistService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+@RestController
```

Comment:
Auto format this file please and use "optimize imports" both options from the menu "Code" in intelliJ